src/libs/logging.ts

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable vue/max-len */
+/* eslint-disable prettier/prettier */
```

Comment:
Is there a reason to disable everything on prettier over what is necessary ?

Reply:
Because of the subtitle file generation, we end up having a lot of lines that would use this same configuration, turning it verbouse, but I think we can activate them within code sessions. Will try.